draft-linus-trans-gossip-ct.md

```diff
@@ -179,20 +181,20 @@ corresponding certificate chain for later use in feedback.
 
 When the client later reconnects to any HTTPS server for the same
 domain it again receives a set of SCTs. The client MUST update its
-store of SCTs for the domain and MUST send to the server the ones in
-the store that were not received from that server.
+store of SCTs [should also discard SCTs from unknown logs] for the domain and MUST send to the server the ones in
+the store that were not received from that server [that are for that server].
```

Comment:
++ to both.  RE: "should also discard SCTs from unknown logs" - Yes, we should be explicit they perform that step again if it is unclear.